README.md

```diff
@@ -3,21 +3,22 @@ GOV.UK Pay Frontend application (Node.js)
+| `SECURE_COOKIE_OFF`                      |            | false/undefined                                                 | To switch off generating secure cookies. Set this to `true` only if you are running self service in a `non HTTPS` environment.                        |
+| `NODE_WORKER_COUNT`                      |            | 1                                                               | The number of worker threads started by node cluster when run in production mode                                                                      |
+| `WORLDPAY_3DS_FLEX_DDC_TEST_URL`         | X          | `https://secure-test.worldpay.com/shopper/3ds/ddc.html`         | URL for Device Data Collection (DDC) initiation in TEST                                                                                               |
+| `WORLDPAY_3DS_FLEX_DDC_LIVE_URL`         | X          |                                                                 | URL for Device Data Collection (DDC) initiation in LIVE                                                                                               |
+| `WORLDPAY_3DS_FLEX_CHALLENGE_TEST_URL`   | X          | `https://secure-test.worldpay.com/shopper/3ds/challenge.html`   | Pointing to Worldpay's TEST 3ds flex challenge URL.                                                                                                   |
+| `WORLDPAY_3DS_FLEX_CHALLENGE_LIVE_URL`   | X          | `https://centinelapi.cardinalcommerce.com/V2/Cruise/StepUp`     | Pointing to Worldpay's LIVE 3ds flex challenge URL.                                                                                                   |
+| `CSP_SEND_HEADER`                        |            | false/undefined                                                 | Apply card payment contest security policy headers.                                                                                                   |
+| `CSP_ENFORCE`                            |            | false/undefined                                                 | Browser will block content security policy violations if set to true, default is to only report on violations.                                        |
+| `CSP_REPORT_URI`                         |            |                                                                 | URI to receive CSP violation reports.                                                                                                                 |
+| `GOOGLE_PAY_MERCHANT_ID`                 |            |                                                                 | Merchant ID used to identified GOV.UK Pay to Google when making a payment request.                                                                    |
```

Comment:
I think you mean “identify” rather than “identified”. Might be useful to put where this comes from (e.g. is it generated by Google? Does it come from some online tool?).